### PR TITLE
Fix: Software templates docs/blog video link

### LINKED ISSUE #23723: 🐛 Bug Report: Software Templates hero video missing
### 📜 Description

the video asset at the top of https://backstage.io/docs/features/software-templates/  is a 404 today.

https://github.com/backstage/backstage/blob/8cc47df993c7cd45eb4d55e8b763f144bcd44c64/docs/features/software-templates/index.md#L9-L16

image

### 👍 Expected behavior

a video! (or removed it!)

## Chrome
image

## Firefox
image

### 👟 Reproduction steps

1. click https://backstage.io/docs/features/software-templates/ 
2. look

### 📃 Provide the context for the Bug.

_No response_

### 🖥️ Your Environment

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

### PR DESCRIPTION
**Pull request by tico88612**
## Hey, I just made a Pull Request!

Fixed #23723

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/features/software-templates/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ code, template in some variables, and then publish the template to some
 locations like GitHub or GitLab.
 
 <video width="100%" height="100%" controls>
-  <source src="https://backstage.io/blog/assets/2020-08-05/feature.mp4" type="video/mp4" />
+  <source src="https://raw.githubusercontent.com/backstage/backstage/master/microsite/blog/assets/2020-08-05/feature.mp4" type="video/mp4" />
```
Comment (awanlin) [inline docs/features/software-templates/index.md:15]: We recently upgraded our Docusaurus version to v3, I'm not totally convinced this is the right direction forward for this. Can you look to see if we can accomplish this using static assets: https://docusaurus.io/docs/static-assets

**Review by awanlin — DISMISSED** [review]
Hi @tico88612, thanks for this, I left a single comment

File: docs/features/software-templates/index.md (hunk 1 above)
Comment (tico88612) [reply docs/features/software-templates/index.md]: If the video needs to be used in both the blog and the docs, a static asset would be a better solution.

**Review by Rugvip — APPROVED** [review]
Thank you! 👍

**Pull request merged**